solana/src/lib.rs

```diff
@@ -1,33 +1,60 @@
-use solana_client::client_error::ClientError;
+use solana_client::{client_error::ClientError, rpc_client::SerializableTransaction};
 use solana_sdk::pubkey::ParsePubkeyError;
-use solana_sdk::signature::Signature;
-use solana_sdk::transaction::Transaction;
-use std::time::SystemTimeError;
+use std::{fs::File, io::Read, path::Path, time::SystemTimeError};
+
+pub use helium_lib::{
+    dao::SubDao,
+    error,
+    keypair::{Keypair, Pubkey, Signature},
```

Comment:
another instance where these seem to be inconsistent between re-exports of the solana_client or solana_sdk types and functions. i can see a good argument to abstract _all_ of the solana stuff behind `helium-lib` and re-exporting all the above stuff there, otherwise i'd use the original source types and only import stuff from helium-lib that's specific to the helium solana ecosystem.